### PR TITLE
update 'Simplified exploration' page

### PR DESCRIPTION
**Pull request by matryer**
* removed extra 'Explore' heading, which is a little confusing in this context
* simplified the intro text to sound more like the rest of the docs

### CODE REVIEW

File: docs/sources/explore/simplified-exploration/_index.md [hunk 1]
```diff
@@ -36,10 +36,8 @@ cards:
 
 # Simplified exploration
 
-Introducing the Grafana Explore apps, designed for effortless data exploration through intuitive, queryless interactions.
+The Grafana Explore apps are designed for effortless data exploration through intuitive, queryless interactions.
 
-Easily explore telemetry signals with these specialized tools, tailored specifically for the Grafana databases to provide quick and accurate insights.
-
-## Explore
+Easily explore telemetry signals with specialized tools, tailored specifically for the Grafana databases to provide quick and accurate insights.
```
Comment (09jvilla) [inline docs/sources/explore/simplified-exploration/_index.md:41]:
```suggestion
Easily explore telemetry signals with these specialized tools, tailored specifically for the Grafana databases to provide quick and accurate insights.
```

Fwiw, I thought the "these" is helpful. But don't feel strongly.

**Review by 09jvilla — APPROVED** [review]

**Comment by 09jvilla** [comment]
Added the "no-changelog" label so the pre-merge check would pass.

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the content!

File: docs/sources/explore/simplified-exploration/_index.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/explore/simplified-exploration/_index.md]: @matryer I think Jen's suggestion is good. I'll add her comments and get this PR merged so we can have the changes live.

**Pull request merged**